src/widgets/vitals/vitals-chart.component.tsx

```diff
@@ -88,27 +88,27 @@ const VitalsChart: React.FC<VitalsChartProps> = ({
   const vitalSigns = [
     {
       id: "bloodPressure",
-      title: `BP (${bloodPressureUnit})`,
+      title: `BP ${bloodPressureUnit ? `(${bloodPressureUnit})` : ""}`,
       value: "systolic"
     },
     {
       id: "oxygenSaturation",
-      title: `SPO2 (${oxygenSaturationUnit})`,
+      title: `SPO2 ${oxygenSaturationUnit ? `(${oxygenSaturationUnit})` : ""}`,
```

Comment:
Introduce:

```ts
function withUnit(label: string, unit: string | null | undefined) {
  return `${label} ${unit ? `(${unit})` : ''}`;
}
```

you can rewrite these to

```js
title: withUnit('SPO2', oxygenSaturationUnit),
```

Reply:
Brilliant, thanks @FlorianRappl. Updated.